docs/reference/esql/functions/types/mv_avg.asciidoc

```diff
@@ -0,0 +1,8 @@
+[%header.monospaced.styled,format=dsv,separator=|]
+|===
+arg1 | result
```

Comment:
I'm changing argument names to *something* thematic as I go. `is_finite` and `is_infinite` are going to go so I'm ignoring them. `n` doesn't seem right for `to_version`. `str` might be better, but I went with `v` because it's a "string formatted like a version" - or, at least, it should be if you want the function to work well.

Reply:
To be clear, I'm not trying to nitpick the chosen names, but was thinking more in the direction of just auto-generating the names to have consistency. In any case, this is not supposed to be an important remark by any means (maybe should have declared that as thought rather than nit).